spec/sideloading_spec.rb

```diff
@@ -337,10 +337,29 @@ def self.name
   describe 'polymorphic_belongs_to macro' do
     let!(:visa) { PORO::Visa.create }
     let!(:mastercard) { PORO::Mastercard.create }
+    let!(:paypal) { PORO::Paypal.create }
     let!(:employee2) do
       PORO::Employee.create credit_card_type: 'Mastercard',
           credit_card_id: mastercard.id
     end
+    let!(:employee3) do
+      PORO::Employee.create credit_card_type: 'Paypal',
+                            credit_card_id: paypal.id
+    end
+    let(:assert_correct_response) do
+      expect(credit_card(0)).to eq({
+                                     'type' => 'visas', 'id' => '1'
+                                   })
+      expect(credit_card(1)).to eq({
+                                     'type' => 'mastercards', 'id' => '1'
+                                   })
+      expect(included[0].jsonapi_type).to eq('visas')
+      expect(included[0].relationships).to eq({
+                                                'visa_rewards' => { 'meta' => { 'included' => false } }
+                                              })
```

Comment:
I know it's minor but I'd prefer these not indent

Reply:
thought I caught them all....